pkg/audit/verifier.go

```diff
@@ -71,12 +72,17 @@ func (verifier *Verifier) Verify(ctx context.Context, stripe *Stripe) (verifiedN
 
 	var offlineNodes storj.NodeIDList
 	var failedNodes storj.NodeIDList
+	var containedNodes storj.NodeIDList
+	var containedPieceNums []int
```

Comment:
Having an array of ints just makes me a little wary since we had some hard-to-find pieceNum/index bugs in the repair package. Can we have a `map[int]storj.NodeID` instead?

Reply:
You are absolutely right. Changed it.